src/js/app.js

```diff
@@ -139,13 +143,19 @@ app.config(['$stateProvider', '$urlRouterProvider', '$locationProvider', '$httpP
         url: '/permissions',
         templateUrl: 'partials/auth/permissions.html',
         controller: 'PermissionCtrl',
-        requireLogin:false
+        requireLogin:false,
+        meta: {
+          'titleSuffix': 'Permissions Dashboard'
+        }
       })
       .state('news', {
         abstract: true,
         url: '/news',
         template: '<ui-view/>',
-        requireLogin: false
+        requireLogin: false,
+        meta {
```

Comment:
You still want to nested the `meta` object map underneath a `data` object map in order for data inheritance via `ui-router` and `ng-meta` to work:
https://github.com/vinaygopinath/ngMeta#data-inheritance-in-ui-router

Also a missing colon here, that's the source of the failing Travis CI build.